cics-bundle-maven-plugin/pom.xml

```diff
@@ -33,20 +33,26 @@
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-archiver</artifactId>
-      <version>3.6.0</version>
+      <version>4.2.7</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-utils</artifactId>
+      <version>3.3.0</version>
     </dependency>
 
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
-      <version>3.5.0</version>
+      <version>3.8.4</version>
```

Comment:
I don't think you can bump the version of the maven core dependency without affecting the minimum version of the maven runtime our plugin will pre-req.  Have you checked that this doesn't affect the minimum version of maven that we are compatible with?

Reply:
I thought it odd myself, so checked with the tool and it seems happy enough. Don't mind going back down though.